docs/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-## 2.6.13 (2018-10-24)
+## 2.6.15 (2018-11-05)
```

Comment:
No need to add version, date ad infos in the changelog. Put your changes in `RELEASE.md` and the build script will insert it in the `CHANGELOG.md`